autogluon/utils/tabular/ml/models/abstract/abstract_model.py

```diff
@@ -153,6 +154,8 @@ def _set_default_auxiliary_params(self):
             # ignore_hpo=False,
             # max_early_stopping_rounds=None,
             # use_orig_features=True,  # TODO: Only for stackers
+            ignored_feature_types_special=None,  # List, drops any features in `self.feature_types_metadata.feature_types_special[type]` for type in `ignored_feature_types_special`. | Currently undocumented in task.
+            ignored_feature_types_raw=None,  # List, drops any features in `self.feature_types_metadata.feature_types_raw[type]` for type in `ignored_feature_types_raw`. | Currently undocumented in task.
```

Comment:
perhaps default = [] is clearer than None that these should be lists.

Reply:
In the future, we may want the ability for different models to eg consider the top k of the provided ngrams rather than ignoring them entirely, so consider adding TODO here.

Reply:
Added both